config/config.go

```diff
@@ -326,10 +326,9 @@ func NewConfig() (c *Config) {
 				Auth: Auth{
 					Type: AuthTypeNone,
 				},
-				Enabled:           false,
+				Enabled:           true,
```

Comment:
Actually it was disabled by default, so the URLs set below had no effect.
But I think we can enable it by default, set the standard istio in-cluster url, and have no external URL by default (so no external link to jaeger).
Does it sound good @aljesusg  ?